subsys/bluetooth/rpc/common/bt_rpc_common.h

```diff
@@ -132,6 +132,10 @@ enum bt_rpc_cmd_from_cli_to_host {
 	BT_ENCRYPT_BE_RPC_CMD,
 	BT_CCM_DECRYPT_RPC_CMD,
 	BT_CCM_ENCRYPT_RPC_CMD,
+#ifdef CONFIG_BT_RPC_INTERNAL_FUNCTIONS
```

Comment:
I am not sure about conditional compilation here, since we might have mess with farther enum values.

Reply:
I have removed the conditional compilation for safety.